components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/typing.scala

```diff
@@ -31,23 +32,31 @@ object typing {
   sealed trait TypingResult {
 
     /**
-     * Checks if there exists a conversion to a given typingResult, with possible loss of precision, e.g. long to int.
-     * If you need to retain conversion precision, use canBeStrictlyConvertedTo
+     * Checks if given type is a target type or given type can be converted to target type without loss of precision
+     * e.g. int to long
```

Comment:
long to int?

Reply:
int to long conversion is acceptable but long to int is not - I precised it